src/otel-collector-agent/Dockerfile

```diff
@@ -9,4 +9,4 @@ ENV OTEL_EXPORTER_OTLP_ENDPOINT=${OTEL_EXPORTER_OTLP_ENDPOINT}
 COPY ./collector.yaml /etc/otel/config.yaml
 
 ENTRYPOINT ["/otelcol-contrib"]
-CMD ["/otelcol-contrib", "--config=/etc/otel/config.yaml"]
+CMD ["/otelcol-contrib", "--config=/etc/otel/config.yaml", "--feature-gates=receiver.hostmetricsreceiver.emitMetricsWithoutDirectionAttribute"]
```

Comment:
What does this do?

Reply:
cumulativetodelta processor is still used, but I got rid of the specific metrics.  Now it converts all monotonic, cumulative sums to monotonic delta sums.